src/main/java/com/revealprecision/revealserver/api/v1/controller/EntityTagController.java

```diff
@@ -0,0 +1,46 @@
+package com.revealprecision.revealserver.api.v1.controller;
+
+import com.revealprecision.revealserver.api.v1.dto.factory.EntityTagResponseFactory;
+import com.revealprecision.revealserver.api.v1.dto.request.EntityTagRequest;
+import com.revealprecision.revealserver.api.v1.dto.response.EntityTagResponse;
+import com.revealprecision.revealserver.service.EntityTagService;
+import io.swagger.v3.oas.annotations.Operation;
+import javax.validation.Valid;
+import lombok.RequiredArgsConstructor;
+import org.springframework.data.domain.Page;
+import org.springframework.data.domain.Pageable;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.MediaType;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RequiredArgsConstructor
+@RestController
+@RequestMapping("/api/v1/entityTag")
+public class EntityTagController {
+
+  private final EntityTagService entityTagService;
+
+  @Operation(summary = "Create Tag", description = "Create Tag", tags = {"Entity Tags"})
+  @PostMapping(consumes = MediaType.APPLICATION_JSON_VALUE, produces = MediaType.APPLICATION_JSON_VALUE)
+  public ResponseEntity<EntityTagResponse> createTag(
+      @RequestBody EntityTagRequest entityTagRequest) {
```

Comment:
what happens on empty and or  null values on EntityTagRequest, please add some validation here.